SeleniumTest/DropdownTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support;
+
+namespace SeleniumTest
+{
+    [TestFixture]
+
+    public class DropdownTest
+    {
+        WebDriver ChromeDriver { get; set; }
+
+        [SetUp]
+        public void SetUp()
+        {
+            ChromeDriver = new ChromeDriver();
+            ChromeDriver.Manage().Window.Maximize();
+            ChromeDriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+        }
+
+        [Test]
+        public void CheckExsistingAllElementsInDropdown()
+        {
+            ChromeDriver.Navigate().GoToUrl("http://the-internet.herokuapp.com/dropdown");
+            IWebElement dropdown = ChromeDriver.FindElement(By.Id("dropdown"));
+            dropdown.Click();
+
+            var elementFirstInDropdown = ChromeDriver.FindElement(By.XPath($"//option[1]"));
+
+            Assert.IsNotNull(elementFirstInDropdown);
+
+            var elementSecondInDropdown = ChromeDriver.FindElement(By.XPath($"//option[2]"));
+
+            Assert.IsNotNull(elementSecondInDropdown);
+        }
+
+        [Test]
+        public void SelectFirstElementInDropdown()
+        {
+            ChromeDriver.Navigate().GoToUrl("http://the-internet.herokuapp.com/dropdown");
+            IWebElement dropdown = ChromeDriver.FindElement(By.Id("dropdown"));
+            dropdown.Click();
+
+            var elementFirstInDropdown = ChromeDriver.FindElement(By.TagName($"option"));
+            elementFirstInDropdown.Click();
+
+            Assert.IsTrue(elementFirstInDropdown.Selected);
+        }
+
+        [Test]
+        public void SelectSecondElementInDropdown()
+        {
+            ChromeDriver.Navigate().GoToUrl("http://the-internet.herokuapp.com/dropdown");
```

Comment:
Each test start from 

ChromeDriver.Navigate().GoToUrl("http://the-internet.herokuapp.com/dropdown");

move it to the setup